apps/ledger-live-desktop/src/renderer/screens/account/index.jsx

```diff
@@ -93,6 +93,16 @@ const AccountPage = ({
 
   const isCompoundEnabled = useCompoundAccountEnabled(account, parentAccount);
 
+  const [banner, setBanner] = useState({});
+
+  useEffect(() => {
+    const fetchData = async () => {
+      const bannerProps = await getBannerProps(account, { t, dispatch });
+      setBanner(bannerProps);
```

Comment:
To prevent a race condition, when we run an async function inside a useEffect, it's good practice to have a killswitch variable before applying the result. In this case it would be  something like 👇🏼 I don't know how time consuming this `getBannerProps` is but it can't harm.
```
useEffect(() => {
    let unmounted = false;
    const fetchData = async () => {
      const bannerProps = await getBannerProps(account, { t, dispatch });
      if (!unmounted) {
        setBanner(bannerProps);
      }
    };
    fetchData();

    return () => {
      unmounted = true;
    }
  }, [account]);
```

Reply:
Good point thanks Juan, but this makes me notice the `getBannerProps` call stack doesn't need to be async so I'll rather make that change instead.